.ibm/pipelines/openshift-ci-tests.sh

```diff
@@ -1,24 +1,27 @@
 #!/bin/sh
 
-set -xe
-export PS4='[$(date "+%Y-%m-%d %H:%M:%S")] ' # logs timestamp for every cmd.
+set -xe  # Enable debugging (-x) and exit on error (-e).
```

Comment:
in another thread in slack we discussed that -x can be dangerous if you're using secrets or tokens in your script. I've not checked this whole script to see if you are using secrets, but generally am not a fan of running a whole script in debug mode unless you have a good reason to do so, as it can leak secrets to the console log.

Reply:
Even though I kind of agree, we need to find a way to keep the upsides of the debugging information being available (it is quite important in such a modular and multi-platform project).
On the next QA weekly we will discuss this topic further.